### ISSUE TITLE
deps probleme

### ISSUE THREAD

**Issue by templine** [eveningkid/denodb#353]
Download https://dev.jspm.io/inherits@2.0
Download https://dev.jspm.io/npm:@jspm/core@1/nodelibs/timers.js
error: Import 'https://dev.jspm.io/inherits@2.0' failed: 500 Internal Server Error
    at https://raw.githubusercontent.com/Zhomart/dex/930253915093e1e08d48ec0409b4aee800d8bd0c/lib-dyn/deps.ts:4:26

**Comment by Codevanger** [comment]
Same issue for me

**Comment by Aiko-Suzuki** [comment]
duplicate of #348

**Comment by watzon** [comment]
I don't think this is a duplicate. dev.jspm.io seems to be broken, and `Zhomart/dex` is attempting imports from dev.jspm.io which are resulting in 500 errors. Even using the branches where #348 is fixed the issue still persists.

**Comment by ninjinskii** [comment]
Check the issue #355, i've made a dirty quick fix, but honestly, i just remove the lib and rewrite my app

**Comment by akatechis** [comment]
I ran into something similar, which I outlined in #359. I am able to import `v1.0.39`, YMMV, in case y'all are depending on something in `v1.0.40`.

**Comment by DevAdriam** [comment]
Same here
